src/common/redis_module/runtest.py

```diff
@@ -72,7 +72,7 @@ def testObjectTableAddAndLookup(self):
   def testObjectTableSubscribe(self):
     p = self.redis.pubsub()
     # Subscribe to an object ID.
-    p.subscribe("object_id1")
+    p.psubscribe("OL:*")
```

Comment:
Not a super big deal, but it'd probably be good to define all of these strings up at the top of the file and save them in variables